spec/config.json

```diff
@@ -975,6 +975,24 @@
           "description": "Sets the refresh token hook endpoint. If set it will be called during token refresh to receive updated token claims.",
           "format": "uri",
           "examples": ["https://my-example.app/token-refresh-hook"]
+        },
+        "authorization_code_hook": {
+          "type": "string",
+          "description": "Sets the authorization code hook endpoint. If set it will be called during token refresh to receive updated token claims.",
+          "format": "uri",
+          "examples": ["https://my-example.app/token-authorization-code-hook"]
+        },
+        "client_credentials_hook": {
+          "type": "string",
+          "description": "Sets the client credentials hook endpoint. If set it will be called during token refresh to receive updated token claims.",
+          "format": "uri",
+          "examples": ["https://my-example.app/token-client-credentials-hook"]
+        },
+        "jwt_bearer_hook": {
+          "type": "string",
+          "description": "Sets the client credentials hook endpoint. If set it will be called during token refresh to receive updated token claims.",
```

Comment:
This needs to be updated - seems like a copy-paste from `client_credentials` section